scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -87,7 +87,7 @@ def net_config(args):
     return get_config(values, fin)
 
 
-def compute_config(args, cpu_flags=cpuflags()):
+def compute_config(args, cpu_flags=cpuflags(), machine="pc-0.14"):
```

Comment:
this is sort of a workaround for being able to run the unit tests on different machines with different cpu's. `cpuflags` and `machine` are options to enable the unittest to pass in an expectation.  @aplanas is there some better way to achieve that?